transactions.go

```diff
@@ -76,7 +76,7 @@ func (t *Transactions) BatchAdd(block BlockID, transactions []Transaction, verif
 
 // BatchUnsafeAdd adds transactions to buffer without adding them to index
 // used on node's start
-func (t *Transactions) BatchUnsafeAdd(txs []*Transaction) {
+func (t *Transactions) BatchUnsafeAdd(txs ...*Transaction) {
```

Comment:
whats the point if we are adding slice here anyways?
`BatchAdd` also works with slice btw

Reply:
It's just to allow for multiple methods of providing a slice of `*Transaction`'s to make tests a little cleaner. Hm, I believe `BatchAdd` in the past used to accept a variadic slice of `*Tranasction`'s, but upon adding in `verifySignature bool` we had to remove it.

Reply:
Yes, I've changed since it was used only with slice. Same here. And even name `Batch` implies that were will be multiple values sent, so slice seems reasonable.

Reply:
Okay, will revert - it makes sense because these batch operations are important to distinguish from single operations (where we lock once vs many times).